policy/reviewer/reviewer.go

```diff
@@ -189,9 +189,16 @@ func FindRandomRequesters(ctx context.Context, prctx pull.Context, result common
 		}
 
 		if len(possibleReviewers) > 0 {
-			logger.Debug().Msgf("Found %d total candidates for review after removing author and non-collaborators; randomly selecting %d", len(possibleReviewers), child.ReviewRequestRule.RequiredCount)
-			randomSelection := selectRandomUsers(child.ReviewRequestRule.RequiredCount, possibleReviewers, r)
-			usersToRequest = append(usersToRequest, randomSelection...)
+			switch child.ReviewRequestRule.Mode {
+			case common.RequestModeAllUsers:
+				logger.Debug().Msgf("Found %d total reviewers after removing author and non-collaborators; requesting all", len(possibleReviewers))
+				usersToRequest = append(usersToRequest, possibleReviewers...)
+			case common.RequestModeRandomUsers:
+			default:
```

Comment:
Should this be the default to conform with current behaviour? Should we bail earlier?

Reply:
If this was the first implementation, I'd pick `all-users` as the default. Given we already released with random selection, let's leave that as the default to preserve compatibility.

Note that cases do not fallthrough by default in Go - use the `fallthrough` keyword. That said, it might be better to fill in a default value earlier so that the `default` case here can be for invalid modes. Take a look at `GetMethods` on the `approve.Options` type for one way to do this. `FillDefaults` on the server configuration is another.